### ISSUE TITLE
TinyMCE does not load if site was added to dock on Mac OS Sonoma

### ISSUE THREAD

**Issue by braendi** [tinymce/tinymce#9083]
**What is the current behavior? Describe the bug**
When I add a website with a TinyMCE component to the dock (https://support.apple.com/en-us/HT213583), and then open it from there, the TinyMCE component does not load. You can check that with your site: https://www.tiny.cloud/tinymce/features/

Looks lik this
![Bildschirmfoto 2023-10-16 um 10 02 54](https://github.com/tinymce/tinymce/assets/13297839/705cd1df-8400-408e-bee2-1b618367e84e)

**Please provide the steps to reproduce and if possible a minimal demo of the problem via [fiddle.tiny.cloud](https://fiddle.tiny.cloud/) or similar.**
In Safari on Mac OS Sonoma, add the site https://www.tiny.cloud/tinymce/features/ to the dock. Then start it from there. The TinyMCE component does not load.

**What is the expected behavior?**
It should load

**Which versions of TinyMCE, and which browser / OS are affected by this issue? Did this work in previous versions of TinyMCE?**
Latest TinyMCE, Safari Browser, Mac OS Sonoma.

**Comment by TheSpyder** [comment]
In that screenshot - which I can replicate - tinymce has started to load, but there's no menubar or toolbar. Is that what you see on other sites as well?

I'm seeing some exceptions in the developer tools but I'd need to run our developer stack to really track down what's going wrong. Thank you for letting us know about this.

**Comment by braendi** [comment]
Yes, that's exactly, what I see on our sites too. A TinyMCE "frame" is there, but it's totally empty.

BTW: how did you start the developer tools? They're deactivated, when I start Safari in "docked" mode.

**Comment by lostkeys** [comment]
I quickly tried making an app out of the [fiddle site](https://fiddle.tiny.cloud/) and it didn't work either. The editor didn't show up at all. I think it's a bummer if when we mac users now got a shiny new native ability to make "app websites" and then TinyMCE is a blocker to making it work.

**Comment by TheSpyder** [comment]
> BTW: how did you start the developer tools? They're deactivated, when I start Safari in "docked" mode.

The same way remote developer tools works for Safari on iphone and ipad - in desktop Safari's `develop` menu an entry will appear for the device, showing every tab available to connect to. The dock app appears as a tab under the current machine "device".

**Comment by braendi** [comment]
Are there any news about this case?

**Comment by braendi** [comment]
Any news?
